src/widget/wrecordingduration.cpp

```diff
@@ -0,0 +1,29 @@
+#include "widget/wrecordingduration.h"
+
+WRecordingDuration::WRecordingDuration(QWidget *parent,
+                                    RecordingManager* pRecordingManager)
+        : WLabel(parent),
+          m_pRecordingManager(pRecordingManager) {
+}
+
+WRecordingDuration::~WRecordingDuration() {
+}
+
+void WRecordingDuration::setup(const QDomNode& node, const SkinContext& context) {
+    WLabel::setup(node, context);
+    connect(m_pRecordingManager, SIGNAL(durationRecorded(QString)),
+        this, SLOT(refreshLabel(QString)));
+    connect(m_pRecordingManager, SIGNAL(isRecording(bool)),
+            this, SLOT(slotReccordingInactive(bool)));
+}
+
+void WRecordingDuration::slotReccordingInactive(bool isRecording) {
+    // If recording is stopped/inactive
+    if(!isRecording) {
+        setText("--:--");
```

Comment:
Our other text widgets provide translatable tooltips, can you add one for ``<RecordingDuration>`` too?